src/frontend/src/pages/CreateChangeRequestPage/CreateProposedSolutionsList.tsx

```diff
@@ -21,10 +20,10 @@ const CreateProposedSolutionsList: React.FC<CreateProposedSolutionsListProps> =
   proposedSolutions,
   setProposedSolutions
 }) => {
-  const auth = useAuth();
+  const user = useCurrentUser();
   const [showEditableForm, setShowEditableForm] = useState<boolean>(false);
 
-  if (!auth.user) return <LoadingIndicator />;
+  if (!user) return <LoadingIndicator />;
```

Comment:
you don't need this check, that's the great part about useCurrentUser, it's never undefined!